API/app/Http/Controllers/Api/ShoppingListController.php

```diff
@@ -177,12 +182,12 @@ public function show(Request $request, $id)
 
         $shoppingList = ShoppingList::findOrFail($id);
 
-        // Authorize user
-        $userId = $request->user_id;
+        // Grab user from sanctum
+        $user = Auth::user();
 
-        if (strcmp($shoppingList->user_id, $userId)) {
+        if (strcmp($shoppingList->user_id, $user->user_id)) {
             // Check if they have permissions for this list in the permissions table
```

Comment:
Append to comment (without quotes): "(read permissions is implied if an entry is present; reading is the bare minimum)"